tests/test_data_cleaning.py

```diff
@@ -1,7 +1,9 @@
-import iup
+import datetime as dt
+
 import polars as pl
 import pytest
-import datetime as dt
+
+import iup
```

Comment:
Again, there's some reordering and extra blank lines here - any reason?

Reply:
See #76